### PR TITLE
Fix run sudo command inside hydra container

### PR DESCRIPTION
**Pull request by aleksbykov**
Any command run inside the hydra container with sudo,
require the password. This cause an issue to install
phantomjs
```
install_phantom_js_script = dedent("""
                sudo rm -rf {phantomjs_base}*
                sudo yum install -y bzip2 fontconfig bitmap-fonts
                curl {phantomjs_url} -o {phantomjs_tar} -L
                tar xvfj {phantomjs_tar}
            """.format(**locals()))
```
and screenshots of grafana were not created and saved in elasticsearch

Next issue happened:
Running 'bash -ce '
sudo rm -rf phantomjs-2.1.1-linux-x86_64*
sudo yum install -y bzip2 fontconfig bitmap-fonts
curl https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 -o phantomjs-2.1.1-linux-x86_64.tar.bz2 -L
tar xvfj phantomjs-2.1.1-linux-x86_64.tar.bz2
''
[sudo] password for abykov: [stderr] 
[stderr] We trust you have received the usual lecture from the local System
[stderr] Administrator. It usually boils down to these three things:
[stderr] 
[stderr]     #1) Respect the privacy of others.
[stderr]     #2) Think before you type.
[stderr]     #3) With great power comes great responsibility.
[stderr] 

Cluster abykov-test-monitor-set-014b5a58 (AMI: ['ami-0436905a94398b3db'] Type: t2.medium): Error taking monitor snapshot: Command 'bash -ce '
sudo rm -rf phantomjs-2.1.1-linux-x86_64*
sudo yum install -y bzip2 fontconfig bitmap-fonts
curl https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 -o phantomjs-2.1.1-linux-x86_64.tar.bz2 -L
tar xvfj phantomjs-2.1.1-linux-x86_64.tar.bz2
'' failed (rc=1)

### CODE REVIEW

**Comment by fruch** [comment]
keep in mind this would prevent us from ever reusing the image between user/accounts.
if those are needed tools, why not just add them into the image ?

**Comment by aleksbykov** [comment]
Israel is right. i can move this command from installation of phantomjs 
```
sudo rm -rf {phantomjs_base}*
sudo yum install -y bzip2 fontconfig bitmap-fonts
```
and move them to  install_prerequistes.sh

**Comment by bentsi** [comment]
yes, please move to `install_prerequistes.sh`

**Comment by aleksbykov** [comment]
Remove sudo commands from installation phantomjs commands

File: docker/env/version [hunk 1]
```diff
@@ -1 +1 @@
-0.31
+0.32
```
Comment (bentsi) [inline docker/env/version]: current is 0.32, please increment

Comment (aleksbykov) [reply docker/env/version]: changed

**Comment by roydahan** [comment]
@aleksbykov / @bentsi do we need backports?
If so, please set lables.

**Pull request merged**